pkgs/development/libraries/glibc/default.nix

```diff
@@ -36,6 +36,7 @@ in
       if [ -f ${stdenv.cc.cc}/lib/libgcc_s.so.1 ]; then
           mkdir -p $out/lib
           cp ${stdenv.cc.cc}/lib/libgcc_s.so.1 $out/lib/libgcc_s.so.1
+          patchelf --set-rpath $out/lib --force-rpath $out/lib/libgcc_s.so.1
```

Comment:
I'd use the usual ``--set-rpath``. The other one is probably just useful when the binaries are to be loaded by an unpatched ld-linux (we change the order when RPATH is considered).